17-evaluation-2/PRESENTATION.md

```diff
@@ -68,43 +67,94 @@ _paginate: false
-English.
+evaluation.
 
-The evaluation will only be **a theoretical part of 60 minutes**.
+The evaluation will last **60 minutes** and the questions will be in French. You
+can answer them in French or English.
 
-The evaluation will be done on the computer with the help the online evaluation
-platform as used in the Evaluation 2.
+The evaluation dates will be as follow:
 
-You will have to be connected to the
-[VPN](https://intranet.heig-vd.ch/services/informatique/poste-de-travail/reseau/vpn/Pages/vpn.aspx).
-It is **mandatory**.
+- DAI-TIC-B (Monday mornings): **25.11.2024**
+- DAI-TIC-C (Friday mornings): **29.11.2024**
 
 ---
 
+The evaluation will be done on paper and is made up of two parts:
+
+1. A first part of 35 minutes
+2. A second part of 25 minutes
+
+A break between the two parts will be given.
+
+## Chapters evaluated
+
 The following chapters will be evaluated:
 
-- [HTTP and curl](https://github.com/heig-vd-dai-course/heig-vd-dai-course/tree/main/21-ssh-and-scp)
-- [Web infrastructures](https://github.com/heig-vd-dai-course/heig-vd-dai-course/tree/main/22-web-infrastructures)
+- [Define an application protocol](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/11-define-an-application-protocol)
+- [Java TCP programming](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/12-java-tcp-programming)
+- [Java UDP programming](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/13-java-udp-programming)
+- [Java network concurrency](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/14-java-network-concurrency)
+- [SMTP and ncat](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/15-smtp-and-ncat)
+- [Practical work 2](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/16-practical-work-2)
+
+![bg right:40%][illustration]
 
 ---
 
-You might be asked questions about the theoretical content, the practical
-content and elements seen in the practical work.
+- [Docker and Docker Compose](https://github.com/heig-vd-dai-course/heig-vd-dai-course/tree/main/06-docker-and-docker-compose)
+  (without the practical content regarding Docker Compose)
+
+![bg right:40%][illustration]
+
+## Evaluation content
+
+You might be asked questions about the theoretical content and the practical
+content in each chapter. You might also be asked about elements seen in the
+practical work as well.
+
+You might also be asked to write/explain code snippets or to explain how to
+solve a specific problem.
+
+![bg right:40%][illustration]
+
+## Notes
 
-You can only have access to your own notes in paper format. Your notes must fit
-on one sheet of paper (recto). They can contain anything useful to you:
-definitions, schemas, code snippets, etc. You will be able to use them during
-the exam at the end of the semester.
+You have access to your own notes in paper format. Your notes must fit on **2
+(1 + 1.5) sheets of paper (recto)**.
```

Comment:
I am not sure I understand this

Reply:
I wanted to illustrate that the students can use 2.5 sheets of paper (the `2` need to be updated, that's a mistake on my side), using the 1 previous note sheet from evaluation 1 + 1.5 new sheets of notes for the new evaluation.

I don't know if there is another way less confusing or do I just drop that part and only leave 2.5 sheets of paper and that's it.